shared/stats/src/index.mjs

```diff
@@ -9,6 +11,21 @@ import { getComponentNames, filterPath, getYaml } from 'govuk-frontend-lib/files
 const componentNamesWithJavaScript = await getComponentNames((componentName, componentFiles) =>
   componentFiles.some(filterPath([`**/${componentName}.mjs`])))
 
+/**
+ * Target files for file size analysis
+ */
+const filesForAnalysis = {
```

Comment:
Considered doing this a bit more automagically, but I think for now just defining the files we want is fine. We can always finesse later.